src/components/molecules/OcBreadcrumb/OcBreadcrumb.vue

```diff
@@ -252,15 +266,19 @@ export default {
 <docs>
 ```js
 <template>
+<section>
   <div>
     <oc-breadcrumb :items="items" class="oc-mb" />
+  </div>
+  <div>
     <oc-breadcrumb :items="items" variation="lead" />
     <oc-breadcrumb :items="items" class="oc-mt-l">
       <template v-slot:contextMenu>
         <p>I'm an example item</p>
```

Comment:
The example list item has a margin top and no other margins or paddings. Was that intended? For the context menu we implemented a `paddingSize` prop in the `OcDrop`. Would be nice to be able to set that for the breadcrumbs as well, if you agree that that could be useful.


<img width="354" alt="Screenshot 2022-01-10 at 22 05 37" src="https://user-images.githubusercontent.com/3532843/148839667-6b876a50-a27e-4004-b025-e56705609c5c.png">

Reply:
You mean a breadcrumb-contextmenu-paddingSize?

Reply:
yep, exactly :-)

Reply:
with the same default as for the context menu in the table